lib/screens/drop_in_checkout/add_payment_method/add_payment_method.dart

```diff
@@ -0,0 +1,258 @@
+
+      final result = await checkout.addPaymentMethod(type: "card", context: context);
+
+      return result;
+    } catch (ex) {
+      //  Handle configuration errors here
+      debugPrint("Initialization Error$ex");
+    }
+    return null;
+  }
+
+  void addPaymentMethod(BuildContext context) async {
+    showProgressIndicator(context);
+    //  Generate order id only once
+    //  Same order id can be reused for dropInCheckout.
+    String? orderId = await prepareOrder();
+    hideProgressIndicator(context);
+    InaiResult result = await inaiAddPaymentMethod(context, orderId!);
+    String resultStr = result.data.toString();
+    String resultTitle = "";
+    switch (result.status) {
+
+      case InaiStatus.success:
+        resultTitle = "Payment Success! ";
+        storePaymentId(result);
+        break;
+
+      case InaiStatus.failed:
+        resultTitle = "Payment Failed!";
+        break;
+
+      case InaiStatus.canceled:
+        resultTitle = "Payment Canceled!";
+        break;
+    }
+    showAlert(context, resultStr, title: resultTitle, callback: () {
+      navigateBackToHome(context);
+    });
+  }
+
+  void storePaymentId(InaiResult result) async {
+    // Obtain shared preferences.
+    SharedPreferences prefs = await SharedPreferences.getInstance();
+    //  Save the new payment method id so we can reuse it later
+    String paymentMethodId = result.data["payment_method_id"];
+    await prefs.setString(paymentMethodIdKey, paymentMethodId);
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: AppBar(
+        title: const Text('Product'),
+        backgroundColor: ThemeColors.bgPurple,
+      ),
+      body: Padding(
+        padding: const EdgeInsets.all(16.0),
+        child: Center(
+            child: Column(
+              mainAxisSize: MainAxisSize.min,
+              children: [
+                Container(
+                  margin: const EdgeInsets.only(top: 15.0),
+                  child: const Image(
+                    image: AssetImage('assets/inai-white.png'),
+                    height: 50,
+                  ),
+                ),
+                Container(
+                  margin: const EdgeInsets.only(top: 15.0),
+                  height: 200,
+                  child: const Image(
+                    image: AssetImage('assets/tshirt.jpeg'),
+                  ),
+                ),
+                Container(
+                    margin: const EdgeInsets.only(top: 15.0),
+                    height: 20,
+                    child: const Text("MANCHESTER UNITED 21/22 HOME JERSEY")),
+                Expanded(
+                  child: Container(
+                    margin: const EdgeInsets.only(top: 15.0),
+                    child: const SingleChildScrollView(
+                      scrollDirection: Axis.vertical,
+                      child: Text('''A FAN JERSEY INSPIRED BY A LEGENDARY HOME KIT.
```

Comment:
can we create localizable file and move the strings into this file ?

Reply:
We are not doing that for any of the SDKs because we want the merchants to be able to use any of these files independently. Creating another file for these strings will create a dependency for this feature on that file.

Reply:
It will be easy if all the strings are handled in one place, also localisation can be achieve easily as all the strings will be in one place .  
@inai-venthan what is your thoughts on this ?